autoware_launch/launch/logging_simulator.launch.xml

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
[Question] What happens if you write it in a simple format like this?

```xml
<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <include file="$(find-pkg-share autoware_launch)/launch/autoware.launch.xml">
    List the parameters to be overridden (no common params such as map_path).
  <include/>
</launch>
```

Reply:
Sorry, I don't understand well.
autoware.launch.xml arguments can be designated when launching logging_simulator.launch.xml? So you want to say we do not have to have arguments in logging_simulator.launch.xml?

Reply:
Exactly, I guess `ros2launch` supports the usage.

Reply:
Got it. Then it would be better with your suggestion. I'll try it.

Some arguments are different, like `launch_map` in autoware.launch.xml and `map` in logging_simulator.lanch.xml.
Do you think it's ok to use `map` argument in autoware.launch.xml?

Reply:
To be precise, I wanted to say three things.
1. Try removing the redundant definitions of args.
2. Try removing the inputs of common args (map_path, etc.).
3. Consider reordering the args. https://github.com/autowarefoundation/autoware_launch/pull/59/files#r918525331

Reply:
> Do you think it's ok to use map argument in autoware.launch.xml?

Hmm? Do you mean some apps are dependent on the arg `map` and we can't change it for now?
If so, I think it's good to define a temporary arg in `logging_simulator.launch` and add a TODO comment to remove it later.

Reply:
I'm not sure if there are some dependent apps. Autoware developers are using `map`, `planning`, `control`, ... arguments in each local environment when launching logging_simualtor, and I'm not sure if I can change these arguments' names.
What do you think.

Reply:
Ah, I see. In that case, I think it's acceptable to define alias args!

Reply:
OK. Then I'll use `map` in logging_simulator.launch.xml and `launch_map` in autoware.launch.xml. Let me know if I misunderstand something.

Reply:
Does your method really work?
I removed group tag and map path (global argument and argument for autoware.launch.xml) in logging_simualtor.launch.xml, but it does not work well.
![image](https://user-images.githubusercontent.com/20228327/178407748-7dcf172c-7a73-411a-9918-dc46e3d4af88.png)

Reply:
Mm, it might be my misunderstanding... I'm sorry. :pleading_face: